ironfish/src/utils/memo.ts

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 function toHuman(memo: string): string {
-  return memo.replace(/[\u0000-\u001F\u007F-\u009F]/g, '').trim()
+  return memo.replace(/[\u0000-\u001F\u007F-\u009F]/gu, '').trim()
 }
```

Comment:
Doesn't this cause it to break, because `\u001F\u007F` will now be interpreted as a single surrogate pair? This needs a test.